public/index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- Import font awesome icon library -->
+    <script src="https://kit.fontawesome.com/490bacaf0b.js" crossorigin="anonymous"></script>
+    <!-- Import Google fonts -->
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Noto+Sans+JP:wght@300&display=swap" rel="stylesheet">
```

Comment:
Could you not import this as normal in CSS? Just maybe better in case someone wants to change the import they can just find it in the CSS file, also it will bundle together I believe so reduces how many imports you have to bring in